src/packages/solid/components/Column/Column.styles.ts

```diff
@@ -14,17 +14,54 @@
  *
  * SPDX-License-Identifier: Apache-2.0
  */
+import type { NodeStyles } from '@lightningjs/solid';
 import theme from 'theme';
+import type { Tone } from 'types';
+import type { ComponentStyleConfig, NodeStyleSet } from '../../types/types.js';
+import { makeComponentStyles } from '../../utils/index.js';
 
-const styles = {
-  Column: {
+export interface ColumnStyle {
+  tone: Tone;
+  Container: NodeStyleSet<{ padding: number[] }>;
+}
+
+type ColumnStyleProperties = {
+  itemSpacing?: NodeStyles['itemSpacing'];
+  itemTransition: NodeStyles['itemTransition'];
+  scrollIndex: NodeStyles['scrollIndex'];
+  gap: NodeStyles['gap'];
```

Comment:
i know it's a little counter intuitive, but these should all be optional. or optionally a Partial